common/server_protocolhandler.cpp

```diff
@@ -373,6 +390,22 @@ void Server_ProtocolHandler::pingClockTimeout()
 
     if (timeRunning - lastDataReceived > server->getMaxPlayerInactivityTime())
         prepareDestroy();
+ 
+    if (QString::fromStdString(userInfo->privlevel()).toLower() == "none")
```

Comment:
![wnb](https://i.kinja-img.com/gawker-media/image/upload/s--sXmOrBZL--/c_fit,fl_progressive,q_80,w_636/1238895886504840879.jpg)

The former can call the latter - for now. One function encapsulating the mechanical check and another function named for the semantic intent.

It's only by current coincidence that those two function names would return the same value. One could change but not the other.